libraries/AP_NavEKF2/AP_NavEKF2.cpp

```diff
@@ -325,7 +325,7 @@ const AP_Param::GroupInfo NavEKF2::var_info[] = {
     // @Param: FLOW_DELAY
     // @DisplayName: Optical Flow measurement delay (msec)
     // @Description: This is the number of msec that the optical flow measurements lag behind the inertial measurements. It is the time from the end of the optical flow averaging period and does not include the time delay due to the 100msec of averaging within the flow sensor.
-    // @Range: 0 250
+    // @Range: 0 127
```

Comment:
It better be, because the parameter is an AP_Int8. We should really have unsigned parameters.